splash/qtrender_png.py

```diff
@@ -6,15 +6,20 @@
 import sys
 import array
 from abc import ABCMeta, abstractmethod, abstractproperty
-from cStringIO import StringIO
+from io import BytesIO
+
 from math import ceil, floor
 
 from PIL import Image
 from PyQt5.QtCore import QBuffer, QPoint, QRect, QSize, Qt
 from PyQt5.QtGui import QImage, QPainter, QRegion
+import six
 
 from splash import defaults
 
+
+xrange = six.moves.range
```

Comment:
Let's not redefine builtins. If values are small & bounded we can use range, if not - six.moves.range.